### ISSUE TITLE
Cant install games ive used the achievement manager on.

### ISSUE THREAD

**Issue by Itsyaboiastro** [gibbed/SteamAchievementManager#330]
I used the manager to get achievements on a game i haven't installed. now I cant install the game. it just says play. I cant do anything with the game

**Comment by gibbed** [comment]
I believe restarting Steam should resolve that.

**Comment by Itsyaboiastro** [comment]
Nope. I've tried a full restart and its still saying its installed and i cant do anything about it.

**Comment by gibbed** [comment]
I don't know what to suggest then, I just tried this myself. Restarted Steam and the game was back to not being uninstalled.

**Comment by Itsyaboiastro** [comment]
> Nope. I've tried a full restart and its still saying its installed and i cant do anything about it.

Managed to fix it.

**Comment by adamtoal** [comment]
how did u fix it

**Comment by Tiramisunel** [comment]
Hi! I seemed to have had the same problem after unlocking some achievements for Resident Evil. The only thing you need to do is uninstall the Steam app, which will also uninstall the games you have, but it will solve the issue. Patience is THE virtue. :)
